src/server/game/Entities/GameObject/GameObjectData.h

```diff
@@ -122,9 +122,9 @@ struct GameObjectTemplate
         {
             uint32 lockId;                                  //0 -> Lock.dbc
             uint32 level;                                   //1
-            uint32 diameter;                                //2 diameter for trap activation
+            uint32 radius;                                  //2 radius for trap activation
```

Comment:
The name in 3.3.5 client is diameter but is used like radius (fix use, don't rename)

Reply:
Well.. thats.. confusing. Thanks